File: 2_fibonacci/fibonacci.go
// Go implementation of the Fibonacci Series
// Author: Eric Alcaide

package main 

import "fmt"

func fibonacci(){
	prev, actual := 0,1

	for q:=0; q < 10; q++{
		actual, prev = prev+actual, actual
		fmt.Println(actual)
	}
}

func main(){
	fibonacci()
}

File: 2_fibonacci/fibonacci.py
# coding: utf8
# Python implementation of the Fibonacci Series
# Author: Eric Alcaide

def fibonacci():
    prev, actual = 0, 1
    
    for q in range(10):
        actual, prev = actual+prev, actual
        print(actual)

def main():
    fibonacci()

if __name__ == "__main__":
    main()
